core/src/main/java/org/apache/iceberg/actions/BaseRewriteDataFilesFileGroupInfo.java

```diff
@@ -21,6 +21,8 @@
 import org.apache.iceberg.StructLike;
 import org.apache.iceberg.relocated.com.google.common.base.MoreObjects;
 
+/** @deprecated will be removed in 1.3.0. */
```

Comment:
I think the deprecation message should mention what should be used alternatively. Also the version might need to be updated (depending on when this gets merged)

Reply:
added the alternative and changed the version to 1.4.0 as 1.2.0 voting is already in progress.